README.md

```diff
@@ -84,7 +84,7 @@ In every event, a `data.id` parameter is defined to allow matching it to the req
 
 ## Compatibility
 
-Requires at least node.js `6.0.0` due to the usage of [ES2015 Proxies](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy). The minimum required `request` version is `2.27.0`, although `2.54.0` is a particularly troubled version which is best avoided.
+Requires at least node.js `4.0.0`. The minimum required `request` version is `2.27.0`, although `2.54.0` is a particularly troubled version which is best avoided.
```

Comment:
The recommended node.js version is `6` as it ships with native [ES2015 Proxy](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy) support. This module supports node.js `4` by means of a polyfill that is loaded under the hood. As usual, keep in mind that polyfills are not as performant as their native counterpart.